doc/developing.rst

```diff
@@ -4,48 +4,65 @@ Developing Reg
 Install Reg for development
 ---------------------------
 
-First make sure you have virtualenv_ installed for Python 3.5.
+.. highlight:: console
 
-.. _virtualenv: https://pypi.python.org/pypi/virtualenv
-
-Now create a new virtualenv somewhere for Reg's development::
+Clone Reg from github::
 
-  $ virtualenv /path/to/ve_reg
+  $ git clone git@github.com:morepath/reg.git
 
-The goal of this is to isolate you from any globally installed
-versions of setuptools, which may be incompatible with the
-requirements of the buildout tool. You should also be able to recycle
-an existing virtualenv, but this method guarantees a clean one.
+If this doesn't work and you get an error 'Permission denied (publickey)',
+you need to upload your ssh public key to github_.
 
-Clone Reg from github (https://github.com/morepath/reg) and go to the
-reg directory::
+Then go to the reg directory::
 
-  $ git clone git@github.com:morepath/reg.git
   $ cd reg
 
-Now we need to run bootstrap.py to set up buildout, using the Python from the
-virtualenv we've created before::
+Make sure you have virtualenv_ installed.
+
+Create a new virtualenv for Python 3 inside the reg directory::
+
+  $ virtualenv -p $(which python3) env/py3
```

Comment:
Maybe you're right. Not sure if this is true for all environments/systems. Remember that I read about some problems. But after digging a bit I found nothing serious and virtualenv itself uses in its tox.ini your way. So I will switch it.